### DISCUSSION TITLE
request - Transaction history per category in the android app.

### DISCUSSION THREAD

**Discussion by Clarke44** [actualbudget/releases#113] [Feature ideas]
Firstly, the app is great. really looking forward to the features in the road map.

For me personally, its nearly 100% there but it would be great to be able to see the the individual transactions for a particular category in the app. 

Following this, it would also be great to be able to cover over spending with other categories without having to do the math? Hopefully that makes sense.

**Comment by MikeBishop** [comment]
This was my wife's primary complaint -- there's no way to see all the activity in a category on mobile.
